tests/Api/Admin/CatalogPromotionsTest.php

```diff
@@ -578,46 +537,41 @@ public function it_updates_catalog_promotion(): void
                     '/api/v2/admin/channels/MOBILE',
                 ],
                 'translations' => ['en_US' => [
-                    '@id' => sprintf('/api/v2/admin/catalog-promotion-translations/%s', $catalogPromotion->getTranslation('en_US')->getId()),
-                    'label' => 'T-Shirts discount',
+                    '@id' => sprintf(
+                        '/api/v2/admin/catalog-promotions/%s/translations/%s',
+                        $catalogPromotion->getCode(),
+                        $catalogPromotion->getTranslation('en_US')->getLocale(),
```

Comment:
It's a bit of an overkill as you are using the code to get the same code 😅 
```suggestion
                        'en_US',
```
the same suggestion applies to other examples below